Sources/SnapshotTesting/Strategy/NSImage.swift

```diff
@@ -111,10 +111,7 @@ private func diff(_ old: NSImage, _ new: NSImage) -> NSImage {
     width: max(old.size.width, new.size.width),
     height: max(old.size.height, new.size.height)
   )
-  let differenceCiImage = differenceFilter.outputImage!
-  let invertFilter = CIFilter(name: "CIColorInvert")!
```

Comment:
Inverting was making it harder to see diffs when images start rendering blank (the difference image looked exactly like the original).

Reply:
oh interesting. we could also do an onion skin style 50% alpha on the two images.

Reply:
Yup, thinkin we can probably refine the diff image in a number of ways.